### PR TITLE
Adding Github templates

### PR DESCRIPTION
**Pull request by rartych**
**What type of PR is this?**

Add one of the following kinds:

* enhancement/feature

**What this PR does / why we need it:**

Github templates will be pre-configured in Repository Template

**Note:**

In order to fully support automatic issue labeling 2 labels need to be added (created): correction , subproject management.
(In Identity and Consent subproject the label colors were set to #DB2F1F and #515C8B respectively)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: 🗣 Subproject discussions 
+    url: https://github.com/camaraproject/Commonalities/discussions
```
Comment (hdamker) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
    url: https://github.com/camaraproject/§repo_name§/discussions
```

File: .github/ISSUE_TEMPLATE/issue_bug_template.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+name: ‼ Bug 🐛
+about: Describe the bug in Provider Implementation
```
Comment (hdamker) [inline .github/ISSUE_TEMPLATE/issue_bug_template.md]:
```suggestion
about: Describe the bug to be fixed
```

(the PR is against lead repository, not Provider Implementation ... not sure if all section below will apply)

Comment (rartych) [reply .github/ISSUE_TEMPLATE/issue_bug_template.md]: As Bug template was intended for Provider Implementation repositories, I removed it from here.
For "bugs" in API specifications the Correction template can be used.

**Review by hdamker — APPROVED** [review]
LGTM

**Pull request merged**